ext/afform/core/CRM/Afform/Upgrader.php

```diff
@@ -88,4 +88,16 @@ public function upgrade_1002(): bool {
     return TRUE;
   }
 
+  /**
+   * Upgrade 1003 - add status column to afform submissions
+   * @see https://lab.civicrm.org/dev/core/-/issues/4232
+   * @return bool
+   */
+  public function upgrade_1003(): bool {
+    $this->ctx->log->info('Applying update 1003 - add status column to afform submissions.');
+    CRM_Core_DAO::executeQuery("ALTER TABLE civicrm_afform_submission ADD status_id INT UNSIGNED NOT NULL  DEFAULT 1 COMMENT 'fk to Afform Submission Status options in civicrm_option_values'");
```

Comment:
I think the upgrade test is failing because of this query. Instead of a direct query you should use the `$this->addColumn()` helper function like this:
https://github.com/civicrm/civicrm-core/blob/f519e1fa8bad0a9e80444f9976afe50547ef27d5/ext/search_kit/CRM/Search/Upgrader.php#L33

Reply:
Thanks, done. All good now :) Ready to merge.